strawberryfields/backends/states.py

```diff
@@ -311,6 +312,56 @@ def poly_quad_expectation(self, A, d=None, k=0, phi=0, **kwargs):
         """
         raise NotImplementedError
 
+    def number_expectation(self, modes):
+        r"""
```

Comment:
Renders like this:
![image](https://user-images.githubusercontent.com/24476053/78277160-08f61f80-74e2-11ea-9982-172deacfc790.png)

Reply:
For all of `BaseState`, `BaseFockState` and `BaseGaussianState`

Reply:
Super cool! Thanks Antal.

Reply:
This looks really nice!